script/openqa-clone-custom-git-refspec

```diff
@@ -21,22 +21,19 @@ Examples:
  openqa-clone-custom-git-refspec -n -c '--show-progress' https://github.com/coolgw/os-autoinst-distri-opensuse/tree/nfs https://openqa.opensuse.org/tests/835060 DESKTOP=textmode
  openqa-clone-custom-git-refspec https://github.com/foursixnine/os-autoinst-distri-opensuse/tree/oopsitsbrokenagain https://openqa.opensuse.org/tests/3128467 NEEDLES_DIR='https://github.com/foursixnine/os-autoinst-needles-opensuse.git#oopsitsbrokenagain'
 
-You need to set the environment variable GITHUB_TOKEN to use authenticated requests.
+To use authenticated requests, you need to either provide a GitHub token via a configuration file at '~/.config/openqa/github-token.conf' or set
```

Comment:
Please wrap at 80 characters. Also can you provide the format of the configuration file?

Reply:
Not sure what else I should provide as the example snippet is all that's necessary 🤔 
Now that I think of it, we could include this into an existing config file like `cllient.conf` tbh. I'll look into that...